cmd/yurt-tunnel-server/app/options/options.go

```diff
@@ -150,7 +150,7 @@ func (o *ServerOptions) Config() (*config.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	cfg.SharedInformerFactory = informers.NewSharedInformerFactory(cfg.Client, 10*time.Second)
+	cfg.SharedInformerFactory = informers.NewSharedInformerFactory(cfg.Client, 24*time.Hour)
```

Comment:
Why do you change the resync time?

Reply:
In my situation, i see thousands tunnel certificate-sign-requests are in pending/approved status. All of them will be enqueued every 10 seconds, which full fills the work-queue. The pending certificate-sign-requests can't be approved. So i did two things:
1. make the re-sync period longer. I think there's no need for the re-sync period to be so short, it's just a method to fix some unexpected situation.  
2. filter before enqueue, only do enqueue for pending tunnel certificate-sign-requests

Reply:
Understood. Alternatively, we may set it to 0 to disable resync .

Reply:
yes, 0 is also ok. let's just make it a relative long period, in case of some unexpected situation.

Reply:
@Fei-Guo Set a relative long period(like 24 * time.Hour) for resync is more reasonable than disabling resync, in case of some unexpected situation.